ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -46,20 +57,33 @@ async function buildWebDriverAgent (xcodeVersion) {
     '--exclude', 'lib',
     '--exclude', 'test',
     '--exclude', 'bundles',
+    '--exclude', 'azure-templates',
   ], {cwd: rootDir});
 
-  // Moved DerivedData/WebDriverAgent-* from Library to uncompressed folder
-  const derivedDataPath = path.resolve(os.homedir(), 'Library', 'Developer', 'Xcode', 'DerivedData');
+  // Move DerivedData/WebDriverAgent-* from Library to "uncompressed" folder
   const wdaPath = (await fs.glob(`${derivedDataPath}/WebDriverAgent-*`))[0];
   await mkdirp(path.resolve(uncompressedDir, 'DerivedData'));
   await fs.rename(wdaPath, path.resolve(uncompressedDir, 'DerivedData', 'WebDriverAgent'));
 
-  // Compress the tarball
-  const pathToTar = path.resolve(pathToBundles, `webdriveragent-xcode_${xcodeVersion}.tar.gz`);
-  env = {COPYFILE_DISABLE: 1};
-  await exec('tar', ['-czf', pathToTar, '-C', uncompressedDir, '.'], {env, cwd: rootDir});
+  // Compress the "uncompressed" bundle as a Zip
+  const pathToZip = path.resolve(pathToBundles, `webdriveragent-xcode_${xcodeVersion}.zip`);
+  await zip.toArchive(
+    pathToZip, {cwd: path.join(rootDir, 'uncompressed')}
```

Comment:
cwd: uncompressedDir

I think these variables could have more descriptive names to make this code more readable